zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -408,11 +394,14 @@ func GetBtcEventWithoutWitness(
 	blockNumber uint64,
 	logger zerolog.Logger,
 	netParams *chaincfg.Params,
-	depositorFee float64,
 ) (*BTCInboundEvent, error) {
-	found := false
-	var value float64
-	var memo []byte
+	var (
+		found        = false
```

Comment:
Warning: I do not know go. 

I understand that := is for declare+assign, while = is for assign. It seems like this changed for defining `found`  here. Is that intended? Yes is 100% an acceptable answer as I truly do not know.